app/hotels/src/filter/hotelFacilities/HotelFacilitiesPopup.js

```diff
@@ -120,17 +121,29 @@ export default class HotelFacilitiesPopup extends React.Component<
     facilitiesListKeys.forEach((key, i) => {
       const facility = facilitiesList[key];
       checkboxes.push(
-        <Checkbox
-          key={i}
-          isChecked={facilities.indexOf(key) >= 0}
-          onPress={this.handleCheckboxOnPress(key)}
-          style={i < facilitiesListKeys.length - 1 && styles.delimiter}
-        >
-          <View style={styles.checkbox}>
-            <Icon name={facility.icon} size={20} style={styles.facilityIcon} />
-            {facility.text}
-          </View>
-        </Checkbox>,
+        <React.Fragment key={i}>
+          <Checkbox
+            isChecked={facilities.indexOf(key) >= 0}
+            onPress={this.handleCheckboxOnPress(key)}
+          >
+            <View style={styles.checkbox}>
+              <TextIcon
+                code={facility.icon}
+                orbit={true}
+                style={styles.facilityIcon}
+              />
+              {facility.text}
+            </View>
+          </Checkbox>
+          {i < facilitiesListKeys.length - 1 && (
```

Comment:
Could you give that a name that is a bit more expressive to understand why the rendering is done or not?